app/views/orders/show.html.haml

```diff
@@ -62,7 +62,7 @@
           %th
             %span.in-grey-600 Total
       %tbody
-        - @order.order_items.each do |order_item|
+        - @_order.order_items.each do |order_item|
```

Comment:
@_order?

Reply:
Add it to rubocop exceptions

Reply:
I use ```@order ``` as instance variable of ApplicationController, so can`t use same name